### PR TITLE
Team Room Rules

### PR DESCRIPTION
**Pull request by KrzysztofCwalina**
It seems like there is still confusion about the morning greetings. Let's try to get the rules of the road on paper.

### CODE REVIEW

File: docs/TeamRoomRules.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+:white_check_mark: **DO** be aware that it might feel awkward in the first week on the team, 
+but the awkwardness goes away once you realize it's similar to saying "how are you doing?", 
+where you don't actually mean what you say.
+
+:ballot_box_with_check: **CONSIDER** exposing coworkers to diverse cultures we celebrate at Microsoft. 
+For example, you can substitute "my friends!" (or even the whole greeting) with "mis amigos!", "moi przyjaciele", etc.
+
+:no_entry: **DO NOT** try to sneak in without the greeting, even if your corowkers are "in the zone" glued to their computers.
+Studies have shown that the small lost of productivity from interrupting your coworkers 
+is ofset by an increase of prodsuctivity due to better team cohesion.
```
Comment (bterlson) [inline docs/TeamRoomRules.md]: *offset, *productivity

File: docs/TeamRoomRules.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+## Responding to a Greeting
+
+:white_check_mark: **DO** respond to the greeter appropriatelly. 
+The reply does not have to, but can, end in "my friend", as it is implied.
```
Comment (bterlson) [inline docs/TeamRoomRules.md]: I'm not sure what normative requirements the word "appropriately" implies above. Did I miss a normative reference?

Also, could use an example Greeting Response here.

File: docs/TeamRoomRules.md (hunk 1 above)
Comment (AlexGhiondea) [inline docs/TeamRoomRules.md]: Typo `prodsuctivity` -> `productivity`

File: docs/TeamRoomRules.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+:white_check_mark: **DO** be aware that it might feel awkward in the first week on the team, 
+but the awkwardness goes away once you realize it's similar to saying "how are you doing?", 
+where you don't actually mean what you say.
+
+:ballot_box_with_check: **CONSIDER** exposing coworkers to diverse cultures we celebrate at Microsoft. 
+For example, you can substitute "my friends!" (or even the whole greeting) with "mis amigos!", "moi przyjaciele", etc.
+
+:no_entry: **DO NOT** try to sneak in without the greeting, even if your corowkers are "in the zone" glued to their computers.
+Studies have shown that the small lost of productivity from interrupting your coworkers
```
Comment (AlexGhiondea) [inline docs/TeamRoomRules.md]: Link to studies?

File: docs/TeamRoomRules.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
+## Entering the Team Room
+
+:white_check_mark: **DO** greet coworkers when you enter the room in the morning. 
+Acceptable greetings end with the phrase "my friends!", or equivalent. For example, "Good morning, my friends!"
+
+:white_check_mark: **DO** be aware that it might feel awkward in the first week on the team, 
+but the awkwardness goes away once you realize it's similar to saying "how are you doing?", 
+where you don't actually mean what you say.
+
+:ballot_box_with_check: **CONSIDER** exposing coworkers to diverse cultures we celebrate at Microsoft.
```
Comment (maririos) [inline docs/TeamRoomRules.md]: We should add a section specifically for this, and every one can keep expanding it.

| Tables        | Are           | 
| ------------- |:-------------:| 
| English      | Good morning my friends |
| Spanish      | Buenos días mis amigos | 
| Polish | ... oi przyjaciele   |

Comment (bterlson) [reply docs/TeamRoomRules.md]: Such a good idea, +1!

This raises a couple questions, though:

1. Is support required for all languages in the table? An alternative would be to define say English as the minimum interoperability requirement with a team-room-implementation-defined list of language transport extensions? But I kind of like the idea of requiring implementations to support at least a few language transports.
2. Should we specify the greeting exchange protocol more concretely? I'm not sure reading the text if it's required/expected that the reply frame use the same language transport as the greeting frame.

File: docs/TeamRoomRules.md (hunk 3 above)
Comment (bterlson) [reply docs/TeamRoomRules.md]: Studies have shown people don't read normative references so I'm not sure this is worth the effort.

File: docs/TeamRoomRules.md (hunk 1 above)
Comment (KrzysztofCwalina) [reply docs/TeamRoomRules.md]: will fix. thanks!

Comment (KrzysztofCwalina) [reply docs/TeamRoomRules.md]: will fix. thanks!

File: docs/TeamRoomRules.md (hunk 3 above)
Comment (KrzysztofCwalina) [reply docs/TeamRoomRules.md]: @AlexGhiondea, I like how you don't let baseless claims drive what we do here on the team. That's why our interview process focuses on data and experiments. 

I will add a link to [Journal of Applied Behavior Analysis paper](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1885415/) I based my assertion on.

File: docs/TeamRoomRules.md (hunk 4 above)
Comment (bterlson) [reply docs/TeamRoomRules.md]: Latin is deprecated so I'd avoid that personally. Esperanto seems fine to me.

File: docs/TeamRoomRules.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
+## Entering the Team Room
+
+:white_check_mark: **DO** greet coworkers when you enter the room in the morning. 
+Acceptable greetings end with the phrase "my friends!", or equivalent. For example, "Good morning, my friends!"
```
Comment (johanste) [inline docs/TeamRoomRules.md]: It is also acceptable to swap out "my friends (or equivalent)" with "I bring fresh donuts".

File: docs/TeamRoomRules.md (hunk 2 above)
Comment (KrzysztofCwalina) [reply docs/TeamRoomRules.md]: This is a very complex subject out of the scope of these guidelines ... but, if you'd like to read more on what's appropriate, feel free to refer to the following: https://nlp.stanford.edu/pubs/politeness.pdf. 

Please note that the paper rates "But what’s the good source you have in mind?" at 0.05 on the politeness scale (page 5), i.e. not very polite.

If you are asking specifically about appropriate greeting reply, see page 2344 in http://infonomics-society.org/wp-content/uploads/licej/published-papers/volume-7-2016/Different-Cultures-and-Social-Patterns-Matter-in-English-and-Chinese-Greetings.pdf. For example, "Good morning, Mr Smith" rates very high, and I am sure it many of your coworkers would appreciate such a polity response. Of course, please substitute "Mr Smith" for the name of the person you are talking to (see the global address book if you don't know these).

File: docs/TeamRoomRules.md (hunk 4 above)
Comment (KrzysztofCwalina) [reply docs/TeamRoomRules.md]: Ni havas interkonsenton, mia amiko!

**Comment by lmazuel** [comment]
All, you can't imagine how happy I am to see this discussion. @annatisch can told you how uncomfortable I was on this topic. I wanted to greet everybody, but I wasn't sure american culture was with me on this. Be sure I bring this to Don's team :)

File: docs/TeamRoomRules.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+:white_check_mark: **DO** be aware that it might feel awkward in the first week on the team, 
+but the awkwardness goes away once you realize it's similar to saying "how are you doing?", 
+where you don't actually mean what you say.
+
+:ballot_box_with_check: **CONSIDER** exposing coworkers to diverse cultures we celebrate at Microsoft. 
+For example, you can substitute "my friends!" (or even the whole greeting) with "mis amigos!", "moi przyjaciele", etc.
+
+:no_entry: **DO NOT** try to sneak in without the greeting, even if your corowkers are "in the zone" glued to their computers.
```
Comment (kfarmer-msft) [inline docs/TeamRoomRules.md]: corowkers -> coworkers

File: docs/TeamRoomRules.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+Studies have shown that the small lost of productivity from interrupting your coworkers 
+is offset by an increase of productivity due to better team cohesion.
+
+## Responding to a Greeting
+
+:white_check_mark: **DO** respond to the greeter appropriatelly.
```
Comment (kfarmer-msft) [inline docs/TeamRoomRules.md]: appropriately -> appropriately

**Comment by Petermarcu** [comment]
I love how our team room is operating. Is this PR ready to be closed out? :)

**Comment by KrzysztofCwalina** [comment]
What about new team members? :-)

**Pull request closed without merging**